core/src/main/java/org/movsim/roadmappings/RoadMappingPoly.java

```diff
@@ -157,6 +157,10 @@ public void addSpiral(double s, double x0, double y0, double theta, double lengt
         roadMappings.add(roadMapping);
     }
 
+    public void addSpiral(Geometry geometry) {
+        addSpiral(geometry.getS(), geometry.getX(), geometry.getY(), geometry.getHdg(), geometry.getLength(), geometry.getSpiral().getCurvStart(), geometry.getSpiral().getCurvEnd());
```

Comment:
Notice that there exists already a RoadMappingSpiral.java file but this is only a mock-up which, at present, defines only an arc (whose curvature is the average of the spiral curvatures) rather than a spiral.